### PR TITLE
XpTracker: xp/h reset

### PR DESCRIPTION
**Pull request by yoyo421**
The xp/h will reset after X seconds from the last xp gained (can be configure).
And new right click option for each skill to reset the xp/h manually.

### CODE REVIEW

**Comment by deathbeam** [comment]
Why would anyone want functionality like this? THis will just make XP tracking inacurrate and not trustable.

**Comment by cernodile** [comment]
After X seconds would be useless, but if you could manually reset after going afk for extended periods of time and want to start counting xp/hr without losing total xp gained, I can see the point of it.

File: runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+package net.runelite.client.plugins.xptracker;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+		keyName = "xpTracker",
+		name = "Xp Tracker",
+		description = "Configuration for the xp tracker plugin"
```
Comment (ypperlig) [inline runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java]: The above should only be indented by 1 tab

**Comment by ypperlig** [comment]
Having only the manual reset button would be nice. The timeout doesn't seem necessary.

**Comment by Nightfirecat** [comment]
The general sense I get from this feature PR is that while a manual "Reset xp/hr" button might be wanted by some users, this automatic reset after X time is not. Given the old age of this PR and need of rebase, I am closing it for now. Anyone interested in creating an updated PR for this feature should keep the discussion on this one in mind.

**Pull request closed without merging**